charts/jaeger/templates/agent-ds.yaml

```diff
@@ -53,10 +53,13 @@ spec:
           {{- end }}
           {{- end }}
         env:
-        {{- if not (hasKey .Values.agent.cmdlineParams "reporter.grpc.host-port") }}
-        - name: REPORTER_GRPC_HOST_PORT
-          value: {{ include "jaeger.collector.name" . }}:{{ .Values.collector.service.grpc.port }}
+        {{- if .Values.agent.extraEnv }}
+          {{- toYaml .Values.agent.extraEnv | nindent 10 }}
         {{- end }}
+          {{- if not (hasKey .Values.agent.cmdlineParams "reporter.grpc.host-port") }}
```

Comment:
(nit)
I am not 100% sure, but i feel we should move this line `{{- if not (hasKey .Values.agent.cmdlineParams "reporter.grpc.host-port") }}`  to match with the `if` section above. Right ?
```
        {{- if .Values.agent.extraEnv }}
          {{- toYaml .Values.agent.extraEnv | nindent 10 }}
        {{- end }}
        {{- if not (hasKey .Values.agent.cmdlineParams "reporter.grpc.host-port") }}
          - name: REPORTER_GRPC_HOST_PORT
            value: {{ include "jaeger.collector.name" . }}:{{ .Values.collector.service.grpc.port }}
        {{- end }
```

Reply:
done!